conf/messages

```diff
@@ -1337,39 +1337,40 @@ expenses.removeExpenses.button = Remove expenses
 
 #####################    Student Loans Question messages    #####################
 
-studentLoansQuestion.title.individual = Did you repay any student loan?
-studentLoansQuestion.title.agent = Did your client repay any student loan?
-studentLoansQuestion.heading.individual = Did you repay any student loan while employed by {0}?
-studentLoansQuestion.heading.agent = Did your client repay any student loan while employed by {0}?
+studentLoansQuestion.title.individual = Did you repay any student loans?
```

Comment:
Those changes will break Journey tests and potentially smoke tests. Have those been updated?

Reply:
Journey tests updated and will check for smoke tests